app/_hub/kong-inc/application-registration/index.md

```diff
@@ -0,0 +1,132 @@
+---
+name: Portal Application Registration
+publisher: Kong Inc.
+version: 1.0.x
+
+desc: Allow portal developers to register applications against services
+description: |
+  Applications allow registered developers on Kong Developer Portal to
+  authenticate with OAuth against a Service on Kong. Admins can
+  selectively admit access to Services using this
+  [Application Registration](/enterprise/1.5.x/developer-portal/administration/application-registration) plugin.
+
+  <div class="alert alert-red">WARNING: This functionality is released as a <a href="/enterprise/latest/introduction/key-concepts/#beta">BETA</a> feature and
+  should not be deployed in a production environment.
+  </div>
+
+enterprise: true
+type: plugin
+categories:
+  - authentication
+
+kong_version_compatibility:
+  enterprise_edition:
+    compatible:
+    - 1.5.x
+
+params:
+  name: application-registration
+  service_id: true
+  consumer_id: false
+  route_id: false
+  protocols: ["http", "https", "grpc", "grpcs"]
+  dbless_compatible: no
+  config:
+    - name: auth_header_name
+      required: false
+      default: "`authorization`"
+      description: |
+        The name of the header that is supposed to carry the access token.
+    - name: auto_approve
+      required: false
+      default: "`false`"
+      value_in_examples:
+      description: |
+        If enabled, all new Service Contracts requests are automatically
+        approved. Otherwise, Dev Portal admins must manually approve requests.
+    - name: description
+      required: false
+      default:
+      value_in_examples:
+      description: |
+        Description displayed in information about a Service in the Developer Portal.
+    - name: display_name
+      required: true
+      default:
+      value_in_examples:
```

Comment:
This needs a sample value, right now the parameter appears in the examples but is blank. Same for `mandatory_scope`

Reply:
I just added a sample value

Reply:
mandatory scope defaults to false. I'm trying to get a scope example working but many permutation attempts are not working. will ask devs.

Reply:
It was a zsh issue. My command works in bash. Thanks to @Jlawlzz for helping me troubleshoot. Added scope example in commit ecca567.